frontend/app/views/fragments/form/cardDetailPayPal.scala.html

```diff
@@ -36,6 +34,9 @@ <h3 class="credit-card-note__header">First payment</h3>
                 <h3 class="credit-card-note__header">Cancellation</h3>
                 <p>You can cancel your membership online through your profile page.</p>
             </div>
+            <div class="credit-card-note">
+                <h3 class="credit-card-note__header"><a class="js-change-payment-method" href="#">Change payment method</a></h3>
+            </div>
```

Comment:
This doesn't show up on mobile for some reason, weird... Perhaps there's a CSS rule hiding it somewhere:

<img width="474" alt="payment-method-mobile" src="https://cloud.githubusercontent.com/assets/5131341/21816510/4ad47d64-d758-11e6-905d-69db6aa79816.png">